integration/substrate/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "tsc; ts-mocha -t 20000 *.spec.ts",
-    "build": "solang compile *.sol test/*.sol --target substrate -v"
+    "build": "parallel solang compile -v --target substrate ::: *.sol test/*.sol"
   },
   "author": "Sean Young <sean@mess.org>",
```

Comment:
This really needs updating, you could add your name here (I think it has to be `contributors` rather than `author`.